### ISSUE TITLE
Button focus outline missing in Firefox on Windows

### ISSUE THREAD

**Issue by bjankord** [cerner/terra-core#193]
### Description of Issue
When tabbing focus to `<Button />`, the focus outline seems to be missing in Firefox on Windows.

###System Configuration
Windows 7

#### Component Version
v0.1.0

#### Browser and Version
Firefox: 52

#### Additional Details (optional)
Image from Firefox on Windows
![button-focus-outline-firefox-windows](https://cloud.githubusercontent.com/assets/633148/24636669/9286535a-18a1-11e7-9c8e-df811f56769e.gif)

### Steps to Reproduce the Issue
1. Navigate to http://engineering.cerner.com/terra-core/#/site/button
1. Press tab key to move focus to `<Button />` components

### Expected Outcomes
- `<Button />` to be styled with focus outline

**Comment by bjankord** [comment]
Fixed in #196
